website/app/Http/Controllers/EventsController.php

```diff
@@ -48,9 +51,10 @@ public function store(CreateEventRequest $request)
       $date     = $date === null ? null : new Carbon($date);
       
       Event::create(compact('title', 'date', 'location'));
-      return;
+      return status["Success"];
```

Comment:
You are returning a string here instead of JSON. You should be using response()->json($status) instead. JSON data might be renderred as a string, but is actually passed through networks as a sequence of bytes, so you should use this method to ensure that it is passed properly.